README.md

```diff
@@ -1,2 +1,41 @@
-# pyramid_zipkin-example
-See how much time python services spend on an http request
+# Basic example showing distributed tracing across python apps
+This is an example app where two pyramid (python) services collaborate on an http request. Notably, timing of these requests are recorded into [Zipkin](http://zipkin.io/), a distributed tracing system. This allows you to see the how long the whole operation took, as well how much time was spent in each service.
+
+Here's an example of what it looks like
+<img width="972" alt="zipkin screen shot" src="https://cloud.githubusercontent.com/assets/64215/19347100/bf76a270-9179-11e6-80bb-bd6ccfe7f7f4.png">
+
+This example was ported from similar examples, such as [Spring Boot](https://github.com/openzipkin/sleuth-webmvc-example).
+
+# Implementation Overview
+
+Web requests are served by [Pyramid](http://www.pylonsproject.org/) routes, and tracing is automatically performed for you by [pyramid_zipkin](https://github.com/Yelp/pyramid_zipkin).
+
+# Running the example
+This example has two services: frontend and backend. They both report trace data to zipkin. To setup the demo, you need to start frontend.py, backend.py and Zipkin.
+
+Once the services are started, open http://localhost:8081/
```

Comment:
I would try to put these parts (which URLs to hit, basically) below the parts that have you actually starting the python and zipkin services. That way the tutorial flows linearly as far as the actual commands you have to run. Feel free to drop if you disagree.